defaults/main.yml

```diff
@@ -1,3 +1,21 @@
 ---
-
 aws_cloudtrail_regions: ['{{ aws_region }}']
+
+cloudwatch_event_rules:
+- name: RootLogin
+  description: 'Alert on any root logins to AWS account'
+  target_id: h3help
```

Comment:
Indigo Macaw is public and thus should not contain any company-specific information.